Makefile

```
@@ -12,7 +12,7 @@ build: build-graphite
 	docker build -t ${PROJECT}/${APP}:${VERSION} .
 
 build-graphite:
-	docker run --rm -v $(shell pwd)/build-graphite:/opt/graphite -v $(shell pwd)/graphite:/opt/build -e VERSION=${VERSION} ubuntu:xenial /opt/build/build.sh
+	docker run --rm -v $(shell pwd)/build-graphite:/opt/graphite -v $(shell pwd)/graphite:/opt/build -e VERSION=${e4ccaa2104499bcc8a39a5479b57a4af898bf9a4} ubuntu:xenial /opt/build/build.sh
```

```suggestion
	docker run --rm -v $(shell pwd)/build-graphite:/opt/graphite -v $(shell pwd)/graphite:/opt/build -e VERSION=e4ccaa2104499bcc8a39a5479b57a4af898bf9a4}ubuntu:xenial /opt/build/build.sh
```

wrapping the commit hash id in ${} will result in it being treated as a env name, which will expand to ""